### DISCUSSION TITLE
How am I fitting this type of images into canvas.

### DISCUSSION THREAD

**Discussion by TNNSLAB** [ImageMagick/ImageMagick#2226] [Help] [answered]
I have this command, but it's sizing up the image. i need to not touch the file, just fit it into canvas: 1560px wide

`convert example.png -resize 1560x1040 -background white -gravity center -extent 1560x1040 out.png`

![logo](https://user-images.githubusercontent.com/46636486/86361748-9594b300-bc74-11ea-8d08-cc0558b0992a.jpeg)

**Comment by snibgo** [comment, marked as answer]
I don't understand. Are you saying you don't want to resize the image? Then remove "-resize 1560x1040".

**Comment by TNNSLAB** [comment]
wow that simple ?! thank you @snibgo
